app/src/main/java/com/ncnf/map/ui/MarkerInfoWindowManager.java

```diff
@@ -104,10 +107,17 @@ public View getInfoWindow(Marker marker) {
     }
 
     @Override
-    public View getInfoContents(com.google.android.gms.maps.model.Marker marker) {
+    public View getInfoContents(Marker marker) {
         if (item == null)
             return null;
         renderInfoWindow();
         return markerWindow;
     }
+
+    private void destroyChildFragment(FragmentManager fragmentManager, Fragment fragment, ConstraintLayout feedContainer, OnBackPressedCallback callback){
+        fragmentManager.beginTransaction().remove(fragment).commit();
```

Comment:
Note : if you want to save a fragment you can add a tag to it and add it to the backstack